src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -713,32 +718,23 @@ export default abstract class ComparisonStore extends AnalysisStore
     } = {};
     readonly selectedStudyMutationEnrichmentProfileMap = remoteData({
         await: () => [this.mutationEnrichmentProfiles],
-        invoke: () => {
-            //Only return Mutation profile if any mutation type is selected, otherwise return {}
-            if (
-                _(this.selectedMutationEnrichmentEventTypes)
-                    .values()
-                    .some()
-            ) {
-                // set default enrichmentProfileMap if not selected yet
-                if (_.isEmpty(this._mutationEnrichmentProfileMap)) {
-                    const molecularProfilesbyStudyId = _.groupBy(
-                        this.mutationEnrichmentProfiles.result!,
-                        profile => profile.studyId
-                    );
-                    // Select only one molecular profile for each study
-                    return Promise.resolve(
-                        _.mapValues(
-                            molecularProfilesbyStudyId,
-                            molecularProfiles => molecularProfiles[0]
-                        )
-                    );
-                } else {
-                    return Promise.resolve(this._mutationEnrichmentProfileMap);
-                }
-            } else {
-                return Promise.resolve({});
-            }
+        invoke: async () => {
```

Comment:
put a comment above explaining that this pertains only to the alterations tab.  or move this to be a property of the tab component.